### PR TITLE
Add more flags for automating the build process on u/s and d/s

### PR DESCRIPTION
**Pull request by irosenzw**
Signed-off-by: Ido Rosenzwig <irosenzw@redhat.com>

### CODE REVIEW

**Comment by irosenzw** [comment]
@djzager @rthallisey @tiraboschi @danielBelenky please review

File: hack/build-manifests.sh [hunk 1]
```diff
@@ -78,10 +107,10 @@ for template in $templates; do
 
 	std_out_file="${std_out_dir}/$(basename -s .in $template)"
 	std_out_file=${std_out_file/VERSION/v$CSV_VERSION}
-
 	rendered=$( \
                  ${PROJECT_ROOT}/tools/manifest-templator/manifest-templator \
                  ${BUILD_FLAGS} \
+								 --hco-container-prefix=${HCO_CONTAINER_PREFIX} \
```
Comment (djzager) [inline hack/build-manifests.sh]: nit: spacing

File: pkg/components/components.go [hunk 2]
```diff
@@ -11,40 +11,40 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const name = "hyperconverged-cluster-operator"
```
Comment (djzager) [inline pkg/components/components.go]: :+1:

File: hack/vars [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (djzager) [inline hack/vars]: What's the point of this...to have a sort of config file if you want it?

File: tools/manifest-templator/manifest-templator.go [hunk 4]
```diff
@@ -443,11 +468,24 @@ func main() {
...
+	kubevirtContainerPrefix := flag.String("kubevirt-container-prefix", *containerPrefix, "")
+	cdiContainerPrefix := flag.String("cdi-container-prefix", *containerPrefix, "")
 	cnaContainerPrefix := flag.String("cna-container-prefix", *containerPrefix, "")
 	webuiContainerPrefix := flag.String("webui-container-prefix", *containerPrefix, "")
+	sspContainerPrefix := flag.String("ssp-container-prefix", *containerPrefix, "")
+	nmoContainerPrefix := flag.String("nmo-container-prefix", *containerPrefix, "")
 	imagePullPolicy := flag.String("image-pull-policy", "IfNotPresent", "")
 	inputFile := flag.String("input-file", "", "")
 
+	cnaMultusImage := flag.String("cna-multus-image", "", "")
```
Comment (djzager) [inline tools/manifest-templator/manifest-templator.go]: I think we should at least converse about the recent explosion of parameters to this.

File: hack/build-manifests.sh [hunk 5]
```diff
@@ -5,64 +5,93 @@ set -e
...
+
+# Use 'latest' tag for all the operators images
+USE_LATEST_TAG="${USE_LATEST_TAG:-false}"
 
 (cd ${PROJECT_ROOT}/tools/manifest-templator/ && go build)
 
 function versions {
-    KUBEVIRT_TAG="$(dep status -f='{{if eq .ProjectRoot "kubevirt.io/kubevirt"}}{{.Version}} {{end}}')"
+    # HCO Tag hardcoded to latest
+    HCO_TAG="latest"
```
Comment (tiraboschi) [inline hack/build-manifests.sh]: ?

**Pull request closed without merging**